src/components/Modal/Modal.css

```diff
@@ -3,14 +3,17 @@
 }
 
 .modal-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
   background: rgba(100,100,100,0.8);
   width: 100%;
   height: 100%;
   position: fixed;
   top: 0px;
   left: 0px;
   pointer-events: none;
-  overflow: scroll;
+  overflow: none;
```

Comment:
`overflow: none;` it's not valid property.

```suggestion

```

Reply:
Doh! Not sure where this came from. 🙄